### PR TITLE
update hello world example

### PR DESCRIPTION
**Pull request by wilzbach**
Hey @stonemaster I just had a short pass through the basic tutorial and tried to enhance the readability.

I hope that most changes make sense to you - feel free to comment on them ;-)

### CODE REVIEW

**Comment by stonemaster** [comment]
Thanks a lot! Looks good but give me some time to have a look at it :-)

File: public/content/en/basics.md [hunk 1]
```diff
@@ -305,12 +324,30 @@ the parent's scope
         }
         ...
 
+A special feature of D is the UFCS (Uniform Function Call Syntax (UFCS)).
```
Comment (stonemaster) [inline public/content/en/basics.md]: There is already a section about UFCS in the "D's Gems" section and I would like to have it there to not introduce too much language features in the basics chapter. That means that we shouldn't use UFCS in the code examples until it has been introduced.

**Comment by wilzbach** [comment]
> Thanks a lot! Looks good but give me some time to have a look at it :-)

Addressed all comments. Sorry for that it took me so long.

**Comment by wilzbach** [comment]
Should I split this off in smaller bits, so that it is easier to merge?

**Comment by stonemaster** [comment]
@wilzbach That would be great! I just made a quick passthrough and found some smaller things but other parts perfectly fine to merge already.

**Comment by wilzbach** [comment]
> That would be great! I just made a quick passthrough and found some smaller things but other parts perfectly fine to merge already.

Done, but seems like the rebasing didn't work for some of the PRs. If a PR contains two commits, that's a mistake. Will fix them later today ;-)

**Comment by wilzbach** [comment]
(this PR is rebased too)

**Pull request merged**